pkg/ui/workspaces/cluster-ui/src/statementDetails/planDetails/plansTable.tsx

```diff
@@ -81,7 +90,7 @@ export const planDetailsTableTitles: PlanDetailsTableTitleType = {
       <Tooltip
         style="tableTitle"
         placement="bottom"
-        content={"The execution count for this Plan."}
+        content={"The execution count for this Explain Explain Plan."}
```

Comment:
Double `Explain` here.

Reply:
ops... I'm really trying to explain it :D